src/common/buffer.cc

```diff
@@ -1838,6 +1844,33 @@ ssize_t buffer::list::read_fd(int fd, size_t len)
   return ret;
 }
 
+#ifdef WITH_SEASTAR
```

Comment:
i can move this helper into `src/crimson/common` so we don't need to taint `buffer.h` and `buffer.cc` in this way.

Reply:
We might also consider `src/common/buffer_seastar.cc`.